FND/Accounting/ISO4217-CurrencyCodes.rdf

```diff
@@ -3486,7 +3482,7 @@
 	</owl:NamedIndividual>
 	
 	<owl:NamedIndividual rdf:about="&fibo-fnd-acc-4217;XAG">
-		<rdf:type rdf:resource="&fibo-fnd-pas-pas;PreciousMetalIdentifier"/>
+		<rdf:type rdf:resource="&fibo-fnd-acc-cur;PreciousMetalIdentifier"/>
 		<rdfs:label>XAG</rdfs:label>
 		<skos:definition>one troy ounce of the precious metal identifier for Silver</skos:definition>
```

Comment:
I know it's not a change this time but I just noticed this: it's nonsense to say "one troy ounce of the ...identifier". It should be something like "identifier for the currency whose unit is one troy ounce of the precious metal silver".  Or could just leave the troy ounce aspect to the Currency definition.
Same for the other identifiers.